cartographer_ros/cartographer_ros/node.h

```diff
@@ -188,6 +188,7 @@ class Node {
   ::ros::Publisher trajectory_node_list_publisher_;
   ::ros::Publisher landmark_poses_list_publisher_;
   ::ros::Publisher constraint_list_publisher_;
+  ::ros::Publisher tracked_pose_in_map_publisher_;
```

Comment:
One small thing: Since the topic is called `tracked_pose`, I would simplify the publisher name to `tracked_pose_publisher`.

Reply:
done